src/crypto/CHIPCryptoPAL.h

```diff
@@ -512,6 +513,54 @@ class AesCcm128Key
     uint8_t bytes[kAES_CCM128_Key_Length];
 };
 
+using AesKeyByteArray = uint8_t[CHIP_CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES];
```

Comment:
This should be called Aes128KeyByteArray

Reply:
I'm not referring to 128 anywhere in the definition. `CHIP_CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES` happens to be 128, but it could be any key size used by the current Matter implementation. So for me 128 is just not needed, but I can change it if it makes things clearer.

Reply:
Ad 1. I envision it in such a way:
 - AesKeys are put it `CryptoContext` of each session.
 - CASE session implementation uses crypto primitives, such as ECDH + HKDF, to securely derive the AesKeys at time of CASE
 - If an application wants to create sessions in other scenarios, it can initialize the `CryptoContext` in an arbitrary way - load keys from non-secure storage or assign known key IDs to the AesKeys. I'm not removing any existing option, I only add the option to use key IDs.
Ad 2. Currently, only one type of key is used, so there's no need to indicate that in the structure. If the support for other key types is added, we will need to probably extend the opaque structure to secure space for encoding the key type. How this is encoded should be platform-specific as platforms that use key IDs will have the key type defined on the secure side.